demo/text_matching/predict.py

```diff
@@ -0,0 +1,125 @@
+# coding:utf-8
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Fine-tuning on text matching task """
+
+import argparse
+import ast
+import paddlehub as hub
+
+# yapf: disable
+parser = argparse.ArgumentParser(__doc__)
+parser.add_argument("--num_epoch", type=int, default=3, help="Number of epoches for fine-tuning.")
+parser.add_argument("--use_gpu", type=ast.literal_eval, default=True, help="Whether use GPU for fine-tuning, input should be True or False")
+parser.add_argument("--checkpoint_dir", type=str, default=None, help="Directory to model checkpoint")
+parser.add_argument("--max_seq_len", type=int, default=512, help="Number of words of the longest seqence.")
+parser.add_argument("--batch_size", type=int, default=32, help="Total examples' number in batch for training.")
+parser.add_argument("--network", type=str, default=None, help="Pre-defined network which was connected after module.")
+args = parser.parse_args()
+# yapf: enable.
+
+if __name__ == '__main__':
+
+    # Load Paddlehub ERNIE pretrained model
+    module = hub.Module(name="ernie")
+
+    # Pair wise task needs: query, title_left, right_title (3 data)
+    # Point wise task needs: query, title_left (2 data)
+    if args.is_pair_wise:
+        num_data = 3
+    else:
+        num_data = 2
+    inputs, outputs, program = module.context(
+        trainable=True, max_seq_len=args.max_seq_len, num_data=num_data)
+
+    # Tokenizer tokenizes the text data and encodes the data as model needed.
+    # If you use transformer modules (ernie, bert, roberta and so on), tokenizer should be hub.BertTokenizer.
+    # else tokenizer should be hub.CustomTokenizer.
+    tokenizer = hub.BertTokenizer(
+        vocab_file=module.get_vocab_path(), tokenize_chinese_chars=True)
+
+    # Load dataset
+    if args.is_pair_wise:
```

Comment:
The same as last comments, if you separate two demo, the dataset is also simple

Reply:
Done